server/channels/store/storetest/channel_store_categories.go

```diff
@@ -27,6 +30,13 @@ func TestChannelStoreCategories(t *testing.T, rctx request.CTX, ss store.Store,
 	t.Run("DeleteSidebarCategory", func(t *testing.T) { testDeleteSidebarCategory(t, rctx, ss, s) })
 	t.Run("UpdateSidebarChannelsByPreferences", func(t *testing.T) { testUpdateSidebarChannelsByPreferences(t, rctx, ss) })
 	t.Run("SidebarCategoryDeadlock", func(t *testing.T) { testSidebarCategoryDeadlock(t, rctx, ss) })
+	for i := 0; i < 2; i++ {
+		i := i
+		t.Run(fmt.Sprintf("SidebarCategoryConcurrentAccess%d", i), func(t *testing.T) {
+			t.Parallel()
+			testSidebarCategoryConcurrentAccess(t, rctx, ss, s)
+		})
+	}
```

Comment:
Whatever we do, we should do it from inside the test. Let's just keep it simple and control the concurrency from inside the test.